README.md

```diff
@@ -19,11 +19,11 @@ Yep! Macmoji substituions are ordinary text substituions. Double click an entry
 
 Some fun recommendations:
 
-| replace | with |
-| :------- | :------- |
-| ehh | ¯\_(ツ)_/¯ |
-| tflip | (╯°□°）╯︵ ┻━┻ |
-| eml | your@email.com |
+| replace  | with           |
+| :------- | :------------- |
+| ehh      | ¯\\\_(ツ)_/¯   |
```

Comment:
ooops just did the same at https://github.com/warpling/Macmoji/pull/5

I think you should also escape the 2nd `_`